propka/__init__.py

```diff
@@ -18,3 +18,7 @@
            "hybrid36", "iterative", "input", "lib", "ligand_pka_values",
            "ligand", "molecular_container", "output", "parameters",
            "protonate", "run", "vector_algebra", "version"]
+
+from ._version import get_versions
+__version__ = get_versions()['version']
+del get_versions
```

Comment:
Why do you need to delete `get_versions` here?

Reply:
This is versioneer’s boiler-plate code. It’s done whenever versioneer adds itself. If you want to keep `get_versions()` you can but I think generally it’s thought that one wants to keep the name space clean and you just access the version as `propka.__version__`. If you need get `get_versions()` for something specific you could use it from `propka._version.get_versions()`. 

Also, not having it exposed allows us to not having to test it explicitly and treat it as implementation detail.

Reply:
Thanks.  I was mainly just curious.